Shared/Samples/Query with time extent/QueryWithTimeExtentView.swift

```diff
@@ -0,0 +1,92 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct QueryWithTimeExtentView: View {
+    /// A map with an ocean basemap style.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISOceans)
+        
+        // Sets the initial viewpoint of the map.
+        map.initialViewpoint = Viewpoint(latitude: 20.0, longitude: -55.0, scale: 9e7)
+        
+        return map
+    }()
+   
+    /// A feature table of Atlantic hurricanes.
+    let featureTable: ServiceFeatureTable = {
+        let featureTable = ServiceFeatureTable(url: .hurricanesService)
+        
+        // Sets the feature request mode to manual (only manual is currently
+        // supported). In this mode, you must manually populate the table -
+        // panning and zooming won't request features automatically.
+        featureTable.featureRequestMode = .manualCache
+        
+        return featureTable
+    }()
```

Comment:
It doesn't make a difference for this sample, but using a `let` constant on a view may be expensive in a view that recomputes. Since the feature table is only used once, consider making it a local variable, such as

```swift
let featureTable = ServiceFeatureTable(url: .hurricanesService)
featureTable.featureRequestMode = .manualCache
let layer = FeatureLayer(featureTable: featureTable)
map.addOperationalLayer(layer)
try await populateFeatures(from: featureTable)
```

Reply:
Can be passed in as a parameter, `private func populateFeatures(from featureTable: ServiceFeatureTable) async throws`